### PR TITLE
[TACHYON-935] Added the usage of links in the documentation in the tachyon.client.block package

### PR DESCRIPTION
**Pull request by HorizonNet**
https://tachyon.atlassian.net/browse/TACHYON-935

### CODE REVIEW

File: clients/unshaded/src/main/java/tachyon/client/block/BlockMasterClientPool.java [hunk 1]
```diff
@@ -23,8 +23,8 @@
 
 /**
  * Class for managing block master clients. After obtaining a client with
- * {@link ResourcePool#acquire}, {@link ResourcePool#release} must be called when the thread is done
- * using the client.
+ * {@link ResourcePool#acquire()}}, {@link ResourcePool#release(Object)}} must be called when the
```
Comment (aaudiber) [inline clients/unshaded/src/main/java/tachyon/client/block/BlockMasterClientPool.java]: extra `}`?

File: clients/unshaded/src/main/java/tachyon/client/block/TachyonBlockStore.java [hunk 2]
```diff
@@ -65,7 +65,7 @@ private TachyonBlockStore() {
    * Gets the block info of a block, if it exists.
    *
    * @param blockId the blockId to obtain information about
-   * @return a FileBlockInfo containing the metadata of the block
+   * @return a {@link tachyon.thrift.FileBlockInfo} containing the metadata of the block
```
Comment (aaudiber) [inline clients/unshaded/src/main/java/tachyon/client/block/TachyonBlockStore.java]: not from your PR, but s/FileBlockInfo/BlockInfo/

**Comment by aaudiber** [comment]
Thanks for the improvements! I've left a few comments

**Comment by HorizonNet** [comment]
@aaudiber Updated the PR to address your comments.

**Comment by aaudiber** [comment]
LGTM

**Pull request merged**